Subject:       Use different header location for oneTBB when using tbb::split
Patch author:  Orson Teodoro <orsonteodoro@hotmail.com>
Date:          Wed Jun 30 08:43:49 PM PDT 2021 (Unix timestamp: 1625111029)

Copyright notice for nanovdb/util/Range.h:

// Copyright Contributors to the OpenVDB Project
// SPDX-License-Identifier: MPL-2.0

You must replace __NANOVDB_USE_ONETBB__ with 0 or 1.

diff -urp nanovdb.orig/util/Range.h nanovdb/util/Range.h
--- nanovdb.orig/util/Range.h	2021-10-29 19:55:40.000000000 -0700
+++ nanovdb/util/Range.h	2022-07-10 19:11:18.417163177 -0700
@@ -16,8 +16,13 @@
 
 #include <cassert>
 
+#define NANOVDB_USE_ONETBB __NANOVDB_USE_ONETBB__		// oteodoro - added line
 #ifdef NANOVDB_USE_TBB
-#include <tbb/blocked_range.h>// for tbb::split
+  #if NANOVDB_USE_ONETBB					// oteodoro - added line
+    #include "tbb/blocked_range.h"// for tbb::split		// oteodoro - reindent line
+  #else								// oteodoro - added line
+    #include "tbb/tbb_stddef.h"// for tbb::split		// oteodoro - re-added line
+  #endif							// oteodoro - indent
 #endif
 
 namespace nanovdb {
